config.schema.json

```diff
@@ -72,7 +72,7 @@
                                     {
                                         "type": "object",
                                         "properties": {
-                                            "packagePath": {
+                                            "path": {
                                                 "type": "string",
                                                 "pattern": "^[A-Za-z0-9._\\-]*$"
```

Comment:
If I parse this correctly, in path we can't have a regex?

Reply:
To our knowledge, no, you must use a predefined path. We could not seem to find anything on their wiki that indicates that this is possible.